terragrunt/modules/rustc-ci/artifacts.tf

```diff
@@ -0,0 +1,187 @@
+locals {
+  rustc_builds     = "rustc-builds"
+  rustc_builds_alt = "rustc-builds-alt"
+  iam_prefix       = "rustc-ci--rust-lang--${var.repo}"
```

Comment:
This iam_prefix should guarantee that the iam stuff doesn't conflict with the existing one in the legacy account